backend/src/v5/processors/teamspaces/projects/models/commons/tickets.js

```diff
@@ -17,18 +17,17 @@
 
 const { addTicket, getAllTickets, getTicketById, updateTicket } = require('../../../../../models/tickets');
 const { basePropertyLabels, modulePropertyLabels, presetModules, propTypes } = require('../../../../../schemas/tickets/templates.constants');
-const { getFileWithMetaAsStream, storeFile } = require('../../../../../services/filesManager');
+const { getFileWithMetaAsStream, removeFile, storeFile } = require('../../../../../services/filesManager');
 const { TICKETS_RESOURCES_COL } = require('../../../../../models/tickets.constants');
 const { generateUUID } = require('../../../../../utils/helper/uuids');
-const { removeFile } = require('../../../../../services/filesManager');
 
 const Tickets = {};
 
 const processBinaryProperties = (template, oldTicket, updatedTicket) => {
 	const toRemove = [];
 	const toAdd = [];
 
-	const updateReferences = (templateProperties, oldProperties = {}, updatedProperties) => {
+	const updateReferences = (templateProperties, oldProperties = {}, updatedProperties = {}) => {
```

Comment:
Because of module properties.. the update data may not include a module

![image](https://user-images.githubusercontent.com/71635617/192716258-94a51670-3384-46fd-a4e7-d82862275099.png)